src/main/java/ua/softserveinc/tc/config/QuartzConfig.java

```diff
@@ -10,9 +10,6 @@
 import ua.softserveinc.tc.constants.QuartzConstants;
 import ua.softserveinc.tc.util.ApplicationConfigurator;
 
-/**
- * Created by Demian on 03.06.2016.
- */
```

Comment:
This dummy comments are spread through all the project, I think we should get rid of them.